Makefile

```diff
@@ -7,18 +7,22 @@ AWS_ACCOUNT?=unknown
 SRC_FOLDER=src
 PUBLIC_FOLDER=_public
 JAVASCRIPTS_LOC=src/javascripts
+JSON_LOC=src/json
 STYLESHEETS_LOC=src/stylesheets
 IMAGES_LOC=src/images
 REPORTS_FOLDER=reports
 
 # Node module variables
-ESLINT=./node_modules/.bin/eslint
-NODE_SASS=./node_modules/.bin/node-sass
-POSTCSS=./node_modules/.bin/postcss
-UGLIFY_JS=./node_modules/.bin/uglifyjs
-IMAGEMIN=./node_modules/.bin/imagemin
-ONCHANGE=./node_modules/.bin/onchange
-PUG=./node_modules/.bin/pug
+NODE_MODULES=./node_modules
+ESLINT=$(NODE_MODULES)/.bin/eslint
+NODE_SASS=$(NODE_MODULES)/.bin/node-sass
+POSTCSS=$(NODE_MODULES)/.bin/postcss
+UGLIFY_JS=$(NODE_MODULES)/.bin/uglifyjs
+IMAGEMIN=$(NODE_MODULES)/.bin/imagemin
+ONCHANGE=$(NODE_MODULES)/.bin/onchange
+PUG=$(NODE_MODULES)/.bin/pug
+LEAFLET=$(NODE_MODULES)/leaflet/dist/leaflet.js
+JSON=$(NODE_MODULES)/pretty-mini-json/pretty-mini-json.js
```

Comment:
Change order to replicate master

ESLINT=./node_modules/.bin/eslint
IMAGEMIN=./node_modules/.bin/imagemin
ONCHANGE=./node_modules/.bin/onchange
NODE_SASS=./node_modules/.bin/node-sass
POSTCSS=./node_modules/.bin/postcss
PUG=./node_modules/.bin/pug
SVGO=./node_modules/.bin/svgo
UGLIFY_JS=./node_modules/.bin/uglifyjs